qase-javascript-commons/changelog.md

```diff
@@ -1,3 +1,28 @@
+# qase-javascript-commons@2.0.0-beta.7
+
+## What's new
+
+TestOps reporter supports uploading test result in real-time.
+It can be useful for long-running tests, where you want to see the results as soon as they are available.
+You can use chunk size to control the size of the data sent to the Qase.
+
+To enable real-time reporting, set an environment variable before running the tests or use the reporter's configuration:
+
+```bash
+QASE_TESTOPS_CHUNK=1
```

Comment:
wait, does it only work for chunk=1? If not, let's give a more reasonable number.

Reply:
Updated the chacgelog